spec/mocks/MockEmailAdapterWithOptions.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (options = {}) => {
```

Comment:
Does Parse Server require an email adapter to launch?

Reply:
You were right, it didn’t need that, so I’ve removed the entire mail mock function and configuration.